test/evp_test.c

```diff
@@ -1798,31 +1801,30 @@ static int encode_test_run(EVP_TEST *t)
     ENCODE_DATA *expected = t->data;
     unsigned char *encode_out = NULL, *decode_out = NULL;
     int output_len, chunk_len;
-    EVP_ENCODE_CTX *decode_ctx;
+    EVP_ENCODE_CTX *decode_ctx = NULL, encode_ctx = NULL;
```

Comment:
Oops. Travis is barfing on this line. I think this should be `*encode_ctx`.

Reply:
Yes, that's what I suggested. Sorry for overlooking it in the fixup. :-/